packages/x-charts-pro/src/internals/material/components/BaseMenuItem.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import { ChartBaseMenuItemProps } from '../../slots/chartBaseSlotProps';
+
+export function BaseMenuItem(props: ChartBaseMenuItemProps) {
```

Comment:
A copy of Data Grid's `BaseMenuItem` ([source](https://github.com/mui/mui-x/blob/8fc7c02ec4031dd606791cf4c52f7d1e0896b371/packages/x-data-grid/src/material/index.tsx#L349))